src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/CommandHandlers/ProviderApproveCohortCommandHandler.cs

```diff
@@ -1,55 +1,40 @@
-﻿using SFA.DAS.Authorization.Features.Models;
-using SFA.DAS.Authorization.Features.Services;
-using SFA.DAS.CommitmentsV2.Data;
+﻿using SFA.DAS.CommitmentsV2.Data;
 using SFA.DAS.CommitmentsV2.Data.Extensions;
-using SFA.DAS.CommitmentsV2.Domain;
 using SFA.DAS.CommitmentsV2.Domain.Interfaces;
 using SFA.DAS.CommitmentsV2.Messages.Commands;
 using SFA.DAS.CommitmentsV2.Types;
 
-namespace SFA.DAS.CommitmentsV2.MessageHandlers.CommandHandlers
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.CommandHandlers;
+
+public class ProviderApproveCohortCommandHandler(
+    ILogger<ProviderApproveCohortCommandHandler> logger,
+    Lazy<ProviderCommitmentsDbContext> dbContext,
+    IEmailOptionalService emailService)
+    : IHandleMessages<ProviderApproveCohortCommand>
```

Comment:
just curious shouldn't these be named with an underscore prefix such as  _logger, _dbContext and _emailService as are they still effectively private class wide vars?